backend/codesirius/problems/tests/test_tag_api.py

```
@@ -0,0 +1,165 @@
+
+        res = self.client.post(TAG_LIST_URL, payload)
+
+        self.assertEqual(res.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_retrieve_tag_success(self):
+        """Test retrieving a tag successfully."""
+        tag = create_tag()
+
+        res = self.client.get(reverse("tag-retrieve-update-destroy", args=[tag.id]))
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)